code/datums/achievements/antagmastery.dm

```diff
@@ -0,0 +1,112 @@
+/datum/award/achievement/antagmastery
+	category = "Antag Mastery"
+	reward = 400
+
+/datum/award/achievement/antagmastery/on_unlock(mob/user)
+	. = ..()
+	for(var/datum/award/achievement/antagmastery/A in subtypesof(/datum/award/achievement/antagmastery))
+		if(!user.client?.player_details.achievements.get_achievement_status(A))
+			return
+	user.client?.give_award(/datum/award/achievement/antagmastery/super, user)
+
+/datum/award/achievement/antagmastery/super
+	name = "Syndie Supreme"
+	desc = "Possess all other antagonist mastery achievements."
+	database_id = MASTERY_ALLANTAGS
+	reward = 20000
```

Comment:
Lower this to like 2000 at most

Reply:
This achievement is insanely hard to acquire, you need a lot of skill, patience, and sometimes luck to get *every single* antag mastery achievement. The beecoin reward should be commensurate with the effort and difficulty to obtain, and I think that this is fair, even possibly on the low side, for what this achievement takes.

Reply:
Well, you already get 400 with every achievement

Reply:
```suggestion
	reward = 5000
```